plugin/draft_editor/draft_editor.py

```diff
@@ -104,15 +118,11 @@ def get_editor_app() -> ui.App:
     raise RuntimeError("Draft editor is not running")
 
 
-def close_editor(submit_draft: bool):
+def close_editor() -> str:
     global last_draft
     remove_tag("user.draft_editor_active")
     actions.edit.select_all()
     selected_text = actions.edit.selected_text()
     actions.edit.delete()
     actions.app.tab_close()
-    actions.user.switcher_focus_window(original_window)
```

Comment:
Recommendation from @AndreasArvidsson in the community backlog session is that we keep `close_editor` with its current behavior and simply save `last_draft` prior to attempting to focus the original window. It is fine if you want to display a notification that focusing the window failed — that is, move the logic that you have included in `draft_editor_submit/draft_editor_discard` back into `close_editor`.

Reply:
Yep that makes sense, moved the logic into `close_editor` now